### ISSUE TITLE
Document supported modes for `chmod` on Windows clearly

### ISSUE THREAD

**Issue by BYK** [shelljs/shelljs#302]

**Comment by nfischer** [comment]
(to provide context)

This is a follow up to #297. It appears that Windows has poor support for `chmod`-like operations. So our docs should probably reflect the shortcomings of `chmod` on Windows systems.

Links (from @BYK)

 - https://github.com/nodejs/node-v0.x-archive/issues/4812
 - https://stackoverflow.com/questions/23329451/node-file-modes-on-windows/23330010#23330010

**Comment by ariporad** [comment]
@BYK: Would you mind opening a PR for this? It seems like you understand this a lot more than I do.

**Comment by nfischer** [comment]
Yeah, if @BYK has the free time to do this, it would be helpful. I'd still label it as low priority, although I'd say this would probably have to block #297 just since I'm not comfortable with taking out unit tests on windows without first documenting that they're not well supported.

I can also try this out, since I have a Windows 7 machine, but I probably won't get to it until later.
